javagen/src/main/java/com/azure/autorest/model/javamodel/JavaPackage.java

```diff
@@ -178,6 +178,21 @@ protected void addJavaFile(JavaFile javaFile) {
         javaFiles.add(javaFile);
     }
 
+    public void addProtocolExamples(ClientMethod method, MethodGroupClient client, String builderName) {
+        method.getProxyMethod().getExamples().forEach((name, example) -> {
+            String filename = name.replace("_", "");
```

Comment:
You might try this instead `com.azure.autorest.preprocessor.namer.CodeNamer.getTypeName(name)`

Reply:
Done